### PR TITLE
Lenovo tab4 and sentry

### PR DESCRIPTION
**Pull request by weberdever**

### CODE REVIEW

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -3067,6 +3067,12 @@ Lenovo:
     - regex: 'Yoga Tablet([^;/]+) Build'
       model: 'Yoga Tablet$1'
       device: 'tablet'
+    - regex: 'Lenovo TB\-X704F Build/NMF26F'
+      model: 'Lenovo Tab 4 10 Plus'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: would you remove the brand name from the model name? The model should only be `Tab 4 10 Plus`.

File: regexes/device/mobiles.yml [hunk 2]
```diff
@@ -3067,6 +3067,12 @@ Lenovo:
...
+      device: 'tablet'
+    - regex: '(Lenovo TB[0-9]?\-[a-z0-9\-]+) Build'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: Please exclude the brand name from the match by using `(TB[0-9]?\-[a-z0-9\-]+) Build`. There's no need to match Lenovo at all, as this is already done with the base regex for the brand.

**Review by sgiehl — CHANGES_REQUESTED** [review]
Thanks for the PR. Left some comments on stuff that needs to be adjusted.

File: regexes/device/mobiles.yml (hunk 2 above)
Comment (weberdever) [reply regexes/device/mobiles.yml]: +

File: regexes/device/mobiles.yml (hunk 1 above)
Comment (weberdever) [reply regexes/device/mobiles.yml]: \+

**Review by sgiehl — APPROVED** [review]

**Pull request merged**